packages/rocketchat-ui-login/login/form.coffee

```diff
@@ -114,10 +114,11 @@ Template.loginForm.events
 
 			else
 				loginMethod = 'loginWithPassword'
+				emailOrUsername = formData.emailOrUsername.replace /^\s+|\s+$/g, ""
```

Comment:
Maybe we could use the `trim()` from http://epeli.github.io/underscore.string/#in-meteor ?